README.md

```diff
@@ -0,0 +1,171 @@
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li>HTML</li>
+    <li>CSS</li>
+		<li>JavaScript</li>
+  </ul>
+</details>
+
+### Key Features <a name="key-features"></a>
+
+- **ignore node_modules in git with .gitignore file**
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live Demo Link here!](https://jhonatanerosa.github.io/Capstone_International-Festival-of-Yucatecan-Music/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+<li>Web browser like <a href='https://www.google.com/chrome/'>Google Chrome</a></li>
+
+<li><a href='https://git-scm.com/'>Git</a></li>
+
+<li><a href='https://github.com/'>GitHub</a></li>
+
+<li><a href='https://code.visualstudio.com/'>Source Code Editor</a></li>
+
+### Setup
+
+Clone this repository to your desired folder:
+
+<a href='https://github.com/JhonatanErosa/Capstone_International-Festival-of-Yucatecan-Music'>International Festival of Yucatecan Music</a>
+
+
+
+
+### Install
+
+Install this dependencies:
+
+npm install
+
+### Usage
+
+To run the project, execute the following command:
+
+<li>Clone the repo to your pc</li>
+
+<li>Use the extension <a href='https://marketplace.visualstudio.com/items?itemName=ritwickdey.LiveServer'>Live Server
+</a> or open de .html file</li> 
+
+### Run tests
+
+To run tests, run the following command:
+
+<li>HTML linters:</li>
+
+npx hint .
+<li>CSS linters:</li>
+
+npx stylelint "**/*.{css,scss}"
+<li>JavaScript linters:</li>
+npx eslint .
+
+### Deployment
+
+You can deploy this project using:
+
+<a href='https://github.com/'>GitHub</a>
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 👥 Author <a name="authors"></a>
+
+👤 **Jhonatan Ali Caamal Erosa**
+
+- GitHub: [@JhonatanErosa](https://github.com/JhonatanErosa)
+- Twitter: [@erosa_c](https://twitter.com/erosa_c)
+- LinkedIn: [Jhonatan Ali Caamal Erosa](https://www.linkedin.com/in/jhonatan-ali-caamal-erosa-bb3829271/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- [1] **SEO optimization**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project support us with a ⭐️
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+I would like to thank the entire Microverse team.
```

Comment:
- Kindly, give appropriate credit to the author of the original design as mentioned in the [requirement](https://github.com/microverseinc/curriculum-html-css/blob/main/capstone/html_capstone.md) 👍